### PR TITLE
Pablo FE Fixes

### PR DESCRIPTION
**Pull request by saadjhk**
## Issue
*Please replace this line with either a link to the ClickUp issue, or a reference to the GitHub issue.*

## Description
*Please replace this line with info that is not in the ClickUp ticket, eg: "also bumps dependency foo version"*

## Checklist

- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_
- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

File: frontend/apps/pablo/defi/subsquid/auctions/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    if (queryResponse.error) throw new Error(queryResponse.error.message);
+    if (!queryResponse.data)
+      throw new Error(
+        "[fetchInitialBalance] Unable to retreive data from query"
+      );
+
+    const { pabloTransactions } = queryResponse.data;
+    if (!pabloTransactions)
+      throw new Error(
+        "[fetchInitialBalance] Unable to retreive data from query"
```
Comment (Dom-Roth) [inline frontend/apps/pablo/defi/subsquid/auctions/helpers.ts:30]:
```suggestion
        "[fetchInitialBalance] Unable to retrieve data from query"
```
nit: typo

File: frontend/apps/pablo/defi/subsquid/auctions/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+  try {
+    const queryResponse = await queryPoolTransactionsByType(
+      pool.poolId,
+      "ADD_LIQUIDITY"
+    );
+    if (queryResponse.error) throw new Error(queryResponse.error.message);
+    if (!queryResponse.data)
+      throw new Error(
+        "[fetchInitialBalance] Unable to retreive data from query"
```
Comment (Dom-Roth) [inline frontend/apps/pablo/defi/subsquid/auctions/helpers.ts:24]:
```suggestion
        "[fetchInitialBalance] Unable to retrieve data from query"
```

File: frontend/apps/pablo/defi/subsquid/auctions/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  try {
+    const queryResponse = await queryPoolTransactionsByType(
+      pool.poolId,
+      "SWAP"
+    );
+    // map to a function later
+    if (queryResponse.error) throw new Error(queryResponse.error.message);
+    if (!queryResponse.data)
+      throw new Error(
+        "[fetchInitialBalance] Unable to retreive data from query"
```
Comment (Dom-Roth) [inline frontend/apps/pablo/defi/subsquid/auctions/helpers.ts:67]:
```suggestion
        "[fetchInitialBalance] Unable to retrieve data from query"
```

File: frontend/apps/pablo/defi/subsquid/auctions/helpers.ts [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+      );
+
+    const { pabloTransactions } = queryResponse.data;
+    if (!pabloTransactions)
+      throw new Error(
+        "[fetchInitialBalance] Unable to retreive data from query"
```
Comment (Dom-Roth) [inline frontend/apps/pablo/defi/subsquid/auctions/helpers.ts:73]:
```suggestion
        "[fetchInitialBalance] Unable to retrieve data from query"
```

File: frontend/apps/pablo/defi/subsquid/auctions/helpers.ts [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+}> {
+  let totalLiquidity = new BigNumber(0);
+  let totalVolume = new BigNumber(0);
+
+  try {
+    const queryResponse = await queryAuctionStats(pool.poolId);
+    if (queryResponse.error) throw new Error(queryResponse.error.message);
+    if (!queryResponse.data)
+      throw new Error(
+        "[fetchInitialBalance] Unable to retreive data from query"
```
Comment (Dom-Roth) [inline frontend/apps/pablo/defi/subsquid/auctions/helpers.ts:99]:
```suggestion
        "[fetchInitialBalance] Unable to retrieve data from query"
```

File: frontend/apps/pablo/defi/subsquid/auctions/helpers.ts [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+      );
+
+    const { pabloPools } = queryResponse.data;
+    if (!pabloPools)
+      throw new Error(
+        "[fetchInitialBalance] Unable to retreive data from query"
```
Comment (Dom-Roth) [inline frontend/apps/pablo/defi/subsquid/auctions/helpers.ts:105]:
```suggestion
        "[fetchInitialBalance] Unable to retrieve data from query"
```

**Review by Dom-Roth — CHANGES_REQUESTED** [review]

File: frontend/apps/pablo/components/Organisms/liquidity/RemoveForm/index.tsx [hunk 7]
```diff
@@ -73,23 +74,51 @@ export const RemoveLiquidityForm: React.FC<BoxProps> = ({ ...rest }) => {
...
+      quoteAsset
+    ) {
+      const selectedLpAmount = toChainUnits(
+        lpBalance.times(debouncedPercentage / 100)
+      );
+
+      const b = baseAsset.network[DEFAULT_NETWORK_ID].toString();
+      const q = quoteAsset.network[DEFAULT_NETWORK_ID].toString();
+
+      // @ts-ignore
```
[the remark this answers is not in the document]
Comment (saadjhk) [reply frontend/apps/pablo/components/Organisms/liquidity/RemoveForm/index.tsx:93]: typescript doesn't pick up RPCs from @polkadot/api (augmentation) so i have manually disable them

**Comment by saadjhk** [comment]
[cu-37bm0b1] [cu-3703d2d] [cu-2kre956] [cu-2nbhp47] Fixes merged here

**Review by easteregg — APPROVED** [review]

**Review by andresmechali — APPROVED** [review]

**Review by vimukthi-git — APPROVED** [review]

**Review by Dom-Roth — APPROVED** [review]

**Review by Dom-Roth — APPROVED** [review]

**Review by easteregg — APPROVED** [review]

**Review by Dom-Roth — APPROVED** [review]

**Review by andresmechali — APPROVED** [review]

**Review by vimukthi-git — APPROVED** [review]

**Pull request closed without merging**